WalletWasabi.Gui/Shell/Commands/SystemCommands.cs

```diff
@@ -36,7 +33,11 @@ public SystemCommands(CommandIconService commandIconService, AvaloniaGlobalCompo
 				exit);
 
 #pragma warning disable IDE0053 // Use expression body for lambda expressions
-			var lockCommand = ReactiveCommand.Create(() => { Global.UiConfig.LockScreenActive = true; });
+			var lockCommand = ReactiveCommand.Create(() => 
+			{ 
+				var global = Locator.Current.GetService<Global>(); 
+				global.UiConfig.LockScreenActive = true; 
+			});
```

Comment:
the `#pragma warning` around this code can be removed.

Reply:
> * It would be great to double click on a closed wallet in the explorer, and it loads then directly. If we do this, then we can also get rid of the `Load Wallet` tab, because it is redundant. I was confused on the first opening to see two lists of my wallets.
> * the tabs really need to be consolidated into one per wallet, it's a mess right now, and I don't think we can keep the concept of three separate tabs per wallet.
> * different colored tabs per wallet is a nice idea
> 
> **Phenomenal** work @danwalmsley, thank you!!

Will work on that in the morning (late afternoon for you guys)

Could you check with @nopara73 if he likes the idea of pre-loading unloaded wallets in Wallet Explorer, and opening them / closing them perhaps with right-click context menu....

im only not sure if there are some issues when it comes to hardware wallets, and any special edge case wallet types perhaps.

Reply:
@MaxHillebrand this was for you